features/editable/cmake/layout_gcc

```diff
@@ -0,0 +1,15 @@
+[build_folder]
```

Comment:
This file should match the documented one: https://github.com/conan-io/docs/pull/1567/files#diff-430fdd2a2f6acd2e29bda5773563cca4R135

Reply:
only the vs layout is being used there in docs